content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -113,34 +94,32 @@ stages:
         md5: ede2872bedbfe10342bb1c416e2f049f
 ```
 
-**Remember: DVC doesn’t replace Git.** Having codified data
-<abbr>artifacts</abbr> and [pipelines](/doc/start/data-pipelines), regular `git`
-commands are used to create versions (commits, tags, branches, etc.):
+> See [Data Pipelines](/doc/start/data-pipelines) to learn about stages.
 
-> Note that DVC automatically prevents Git from tracking data in the
-> <abbr>workspace</abbr> (via `.gitignore`).
+Having codified the data and models in the project, a regular `git` workflow can
+be used to create versions (commits), branches, etc. using the metafiles as
+proxies to the underlying data:
 
 ```dvc
-$ git add cleanup.sh
+$ git add training.py
 $ dvc add data/
 ...
 $ git add data.dvc data/.gitignore
+$ git commit -m 'First modeling experiment'
 
-$ git commit -m 'Data cleanup v1.0'
+# Iterate, repeat!
 ```
 
-> By matching versions of code with snapshots of the data at the time, we can
-> achieve full [reproducibility](/doc/start/data-pipelines).
+> Note that DVC prevents Git from tracking data in the workspace (via
+> `.gitignore`).
 
-Now that Git is tracking the code (including
-[DVC metafiles](/doc/user-guide/dvc-files-and-directories)), and DVC is tracking
-the data, we can repeat the procedure to generate more commits. `dvc checkout`
-can then be used to go back:
+With the project metadata in Git (along with matching code), it can now easily
+be rewound ⏪:
 
 ```dvc
 $ git checkout v1.0
 $ dvc checkout
-M       data\raw
+M       data
 ```
 
 ![](/img/versioning.png) _Full project restoration_
```

Comment:
Keeping this old image is a bit forced. May not be necessary at all if we make a new figure for the **How it looks & feels** section.